### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cobacoba-ops**
hey i just to add more statment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 learn how to squash<br>
 this is first commit 
 this commit is the last commit for learning squash purpose
+Code management is easier when using github
```
Comment (hwangzie) [inline README.md:5]: you need to use <**br**> tag before adding new statment

**Review by hwangzie — CHANGES_REQUESTED** [review]
need to use the right format

File: README.md (hunk 1 above)
Comment (cobacoba-ops) [reply README.md:5]: h

Comment (hwangzie) [reply README.md:5]: acc

**Review by hwangzie — APPROVED** [review]

**Comment by hwangzie** [comment]
thanks for the colaboration @cobacoba-ops

**Pull request merged**